build/xtask/src/sizes.rs

```diff
@@ -78,7 +79,10 @@ pub fn run(
         }
         let size = toml.kernel.requires[&mem.to_string()];
 
-        let suggestion = toml.suggest_memory_region_size("kernel", used);
+        let suggestion = toml.suggest_memory_region_size("kernel", used, 1);
+        assert_eq!(suggestion.len(), 1);
```

Comment:
So imagine I'm here in the future and this assertion has failed. What should I look at? Asking because I don't totally understand it as a reviewer. (The answer would make a good assert message or comment.)

Reply:
Added a string to the assertion in 7d2d02b919f84713f1902d88ba41c865f5363f92; this detects a failure in `suggest_memory_region_size` where it has suggested > 1 region for the kernel, which should never happen.